lib/tasks/technical_metadata.rake

```diff
@@ -12,8 +14,8 @@ namespace :techmd do
   end
 
   desc 'Generate technical metadata from Moab'
-  task :generate_for_moab, %i[druid] => :environment do |_task, args|
-    MoabProcessingService.process(druid: args[:druid])
+  task :generate_for_moab, %i[druid force] => :environment do |_task, args|
+    MoabProcessingService.process(druid: args[:druid], force: args[:force] == 'true')
```

Comment:
See question above about the string value `'true'`; is this something that is necessary for the rake task to run correctly? (My quick web search turned up nothing conclusive)